spec/requests/api/v1.1/portfolios_controller_spec.rb

```diff
@@ -176,6 +176,17 @@
 
       expect(response).to have_http_status(:no_content)
     end
+
+    context "bad group uuid" do
+      let(:group_uuid) { "fred" }
+
+      it "returns a 400" do
+        subject
+
+        expect(response).to have_http_status(:bad_request)
+        expect(first_error_detail).to match(/OpenAPIParser::InvalidUUIDForm/)
+      end
+    end
```

Comment:
Should we duplicate this spec for the #unshare endpoint spec too?

Reply:
@eclarizio I have added a spec for unshare